src/index.html

```diff
@@ -11,7 +11,220 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="main-form"
+    >
+
+      <fieldset class="block-forms personal-block">
+        <legend class="title">
+          Personal information
+        </legend>
+        <div class="personal-inf">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="personal-inf">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            autocomplete="off"
+          >
+        </div>
```

Comment:
It'll also work. You don't really need this additional wrapper, but it's up to you
```suggestion
          <label class="personal-info">
            Name:
            <input
              type="text"
              name="name"
              id="name"
              required
              autocomplete="off"
            >
          </label>
```